RocketChat/Rocket.Chat.ReactNative
# Chore: Migration to Hooks - FormTextInput

## Proposed changes


## Issue(s)	


## How to test or reproduce
As we changed the FormTextInput, we should test all inputs related: 
* New server view;
* Login;
* Register;
* Forgot password; 
* Change Status;
* Share view;
* Search Message view;
* Room view -> join code;
* Room View -> Edit info;
* Profile;
* E2E screens... (Enter your password)
* Create Discussions;
* Two Factor

## Screenshots

## Types of changes




## Checklist



## Further comments
